File: Medium/14_Longest_Common_Prefix/solution_2.py
class Solution(object):
    def longestCommonPrefix(self, strs):
        """
        :type strs: List[str]
        :rtype: str
        """
        v = sorted(strs)
        first = v[0]
        last = v[-1]

        min_len = min(len(first), len(last))
        ans = ""
        for i in range(min_len):
            if (first[i] != last[i]):
                return ans
            ans+=first[i]
        return ans

File: Easy/58_Length_of_Last_Word/solution_2.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
       int len = s.length();
       int sz=0;
       for(int i=len-1; i>=0; i--)
       {
           if(s[i] != ' ') 
           {
               while(s[i] != ' ')
               {
                   sz++;
                   i--;
                   if( i<0) break;
               }
               break;
           }

       } 
    
        return sz;
    }
};
